Cargo.toml

```diff
@@ -14,6 +14,6 @@ readme = "README.md"
 path = "lib.rs"
 
 [dependencies]
+hostname = "0.3"
```

Comment:
Maybe bump the version of this crate if this is a breaking change?

Reply:
Ok. I pushed another commit to a bump the version.